src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2891,6 +2891,101 @@ void SmartScript::ProcessAction(SmartScriptHolder& e, Unit* unit, uint32 var0, u
             }
             break;
         }
+        case SMART_ACTION_FOLLOW_GROUP:
+        {
+            if (!e.action.followGroup.followState)
+            {
+                for (WorldObject* target : targets)
+                    if (IsUnit(target))
+                        target->ToCreature()->GetMotionMaster()->MoveIdle();
```

Comment:
```suggestion
                for (WorldObject* target : targets)
                {
                    if (IsUnit(target))
                    {
                        target->ToCreature()->GetMotionMaster()->MoveIdle();
                    }
                }
```

Reply:
It is not needed. It hasn't been enforced for a very very long time. And following the code style of the majority of the project this is fine